[AY1920S1-CS2113-T09-1] Hustler

yzia2000 opened the pull request:
@ngjiewu
@nystera
@jingkang97
@EnriqueKhai
@yzia2000

seanieyap commented on src/main/java/seedu/hustler/command/taskCommand/AddCommand.java:
```diff
@@ -0,0 +1,60 @@
+        switch (timeCommand) {
+        case "by":
+            taskType = "deadline";
+            break;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/command/taskCommand/AddCommand.java:
```diff
@@ -0,0 +1,60 @@
+            break;
+        case "at":
+            taskType = "event";
+            break;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/command/taskCommand/AddCommand.java:
```diff
@@ -0,0 +1,60 @@
+            break;
+        default:
+            ui.show_message("/" + timeCommand + " is an invalid addition to /add");
+            return;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/command/taskCommand/DeleteCommand.java:
```diff
@@ -0,0 +1,41 @@
+
+    /**
+     * Deletes task at index taskIndex inside.
+     *
```
Extra line

seanieyap commented on src/main/java/seedu/hustler/command/taskCommand/RemindCommand.java:
```diff
@@ -0,0 +1,20 @@
+    
+    /**
+     * Executes remind pipeline.
+     *
```
Extra line can be deleted

seanieyap commented on src/main/java/seedu/hustler/command/taskCommand/SortCommand.java:
```diff
@@ -0,0 +1,32 @@
+
+    /**
+     * Sorts the task list.
+     *
```
Extra line can be deleted

seanieyap commented on src/main/java/seedu/hustler/data/CommandLog.java:
```diff
@@ -0,0 +1,53 @@
+
+                    Hustler.saveStorage();
+                } catch (CommandLineException | IOException e) {
+
```
??

seanieyap commented on src/main/java/seedu/hustler/data/CommandLog.java:
```diff
@@ -0,0 +1,53 @@
+
+    public static boolean isRestoring() {
+        return isRestoring;
+    }
```
Missing header comments. All non trivial methods should have javadoc format header comments. You have done so for the rest of the classes but not here.

seanieyap commented on src/main/java/seedu/hustler/data/Folder.java:
```diff
@@ -0,0 +1,25 @@
+    /**
+     * Checks directory to see if file exists. If it
+     * doesn't exist, creates one.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/data/MemoryManager.java:
```diff
@@ -0,0 +1,30 @@
+    public void reloadBackup(int numberOfCommandsToUndo) {
+        Hustler.reloadBackup();
+        CommandLog.restoreData(numberOfCommandsToUndo);
+    }
```
Missing header comments. All non trivial methods should have javadoc format header comments.

seanieyap commented on src/main/java/seedu/hustler/data/Storage.java:
```diff
@@ -0,0 +1,219 @@
+
+    /**
+     * Creates a backup copy of hustler.txt each time the user uses this app; used for UndoCommand.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/data/Storage.java:
```diff
@@ -0,0 +1,219 @@
+
+    /**
+     * Reloads list of tasks from a backup of the original hustler.txt file.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/data/Storage.java:
```diff
@@ -0,0 +1,219 @@
+
+    /**
+     * Loads list of tasks from disc from a csv style file.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/game/achievement/AchievementList.java:
```diff
@@ -0,0 +1,128 @@
+     */
+    public static ArrayList<Achievements> achievementList = new ArrayList<>();
+
+
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/game/achievement/Achievements.java:
```diff
@@ -0,0 +1,65 @@
+    /**
+     * Update how much point is gained from unlocking an achievement.
+     * @param points
+     * @return
```
Missing comment details.

seanieyap commented on src/main/java/seedu/hustler/game/achievement/Achievements.java:
```diff
@@ -0,0 +1,65 @@
+    /**
+     * Unlock an achievement.
+     * @param lock
+     * @return
```
Missing comment details.

seanieyap commented on src/main/java/seedu/hustler/game/achievement/Achievements.java:
```diff
@@ -0,0 +1,65 @@
+
+    /**
+     * Format which the achievement will be printed out.
+     * @return
```
Missing comment details.

seanieyap commented on src/main/java/seedu/hustler/game/achievement/Achievements.java:
```diff
@@ -0,0 +1,65 @@
+     * @return
+     */
+    public String toString(){
+        return "Gained:";
```
??

seanieyap commented on src/main/java/seedu/hustler/game/achievement/FirstLogin.java:
```diff
@@ -0,0 +1,134 @@
+
+/**
+ * Achievement which can be attained after user first logs in.
+ *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/game/avatar/Level.java:
```diff
@@ -0,0 +1,110 @@
+
+/**
+ * Class representing the level of the
+ * avatar. Also contains the current xp.
```
Can be on the same line?

seanieyap commented on src/main/java/seedu/hustler/game/avatar/Stats.java:
```diff
@@ -0,0 +1,80 @@
+
+    /**
+     * Increases the stats based on the current level.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/game/avatar/Stats.java:
```diff
@@ -0,0 +1,80 @@
+    /**
+     * Construct Stats with the given damage, defence, stamina
+     * and speed.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/logic/CommandLogic.java:
```diff
@@ -0,0 +1,266 @@
+     *
+     * @param rawInput raw user input
+     * @param userInput split user input
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/logic/CommandLogic.java:
```diff
@@ -0,0 +1,266 @@
+
+    /**
+     * Validates event to check for format.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/game/shop/items/ShopItem.java:
```diff
@@ -0,0 +1,82 @@
+/**
+ * The class in which every item will inherit from.
+ */
+public abstract class ShopItem implements Purchasable {
```
These shopitems may be better programmed as data where by you read item details from some data file instead of managing these items as classes/objects. In the future where this application has hundreds of items, it would be very tough to modify the code base. For instance when adding some metric of an item. But since this is already done, good job.

seanieyap commented on src/main/java/seedu/hustler/parser/DateTimeParser.java:
```diff
@@ -0,0 +1,128 @@
+     * Date and time with format d/M/uuuu HHmm.
+     */
+    private static final String DATE_TIME_FORMATTER = "d/M/uuuu HHmm";
+    /**
```
New line is missing.

seanieyap commented on src/main/java/seedu/hustler/parser/DateTimeParser.java:
```diff
@@ -0,0 +1,128 @@
+    public static String convertDateTime(LocalDateTime localDateTime) {
+
+        String dateTime;
+
```
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.

seanieyap commented on src/main/java/seedu/hustler/parser/DateTimeParser.java:
```diff
@@ -0,0 +1,128 @@
+            break;
+        default:
+            suffix = "th";
+            break;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/parser/DateTimeParser.java:
```diff
@@ -0,0 +1,128 @@
+            break;
+        case 3:
+            suffix = "rd";
+            break;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/parser/DateTimeParser.java:
```diff
@@ -0,0 +1,128 @@
+            break;
+        case 2:
+            suffix = "nd";
+            break;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/parser/DateTimeParser.java:
```diff
@@ -0,0 +1,128 @@
+        switch (dateTime.getDayOfMonth() % 10) {
+        case 1:
+            suffix = "st";
+            break;
```
Indentation problem here.

seanieyap commented on src/main/java/seedu/hustler/parser/ParserForCommand.java:
```diff
@@ -0,0 +1,35 @@
+
+    /**
+     * Initializes command variable.
+     *
```
Extra line can be deleted.

seanieyap commented on src/main/java/seedu/hustler/task/Reminders.java:
```diff
@@ -0,0 +1,235 @@
+    /**
+     * 30 minutes converted to seconds.
+     */
+    private static final int THIRTY_MINUTES = 1800;
```
New lines should seperate these since they have comments

seanieyap commented on src/main/java/seedu/hustler/task/Reminders.java:
```diff
@@ -0,0 +1,235 @@
+                if (checkOverdue(i)) {
+                    overDueList.add(list.get(i));
+                }
+            }
```
These two if statements can be combined into one.

seanieyap commented on src/main/java/seedu/hustler/task/Reminders.java:
```diff
@@ -0,0 +1,235 @@
+    public static void displayReminders() {
+        if (!overDueList.isEmpty()) {
+            if (overDueList.size() == 1) {
+                System.out.println("\t_____________________________________");
```
These lines can be saved into some variable since they are used so many times. Also to avoid "magic strings"; if you need to change these in the future, it would be a hassle to do so to all of them.

seanieyap commented on src/main/java/seedu/hustler/task/Task.java:
```diff
@@ -0,0 +1,149 @@
+    public String toString() {
+        return "[" + this.getStatusIcon() + "]" + this.getDifficulty().toString() +
+                this.tag.toString() + " " +
+            this.getDescription();
```
Should be on the same line

seanieyap reviewed (commented):
Overall good job. Just some coding quality issues like magic numbers and commenting issues.

seanieyap commented:
Please code this PR after reading my comments.

ngjiewu committed:
Update UserGuide.adoc

nystera committed:
+fixed bug to able to parse difficulty.

nystera committed:
+merged.

ngjiewu committed:
Fixed null pointer exception

ngjiewu committed:
Changed completed symbol to + and -

ngjiewu committed:
Merge pull request #250 from ngjiewu/master

Fixed null pointer exception

ngjiewu committed:
Added commands to taskCompletionMode

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

nystera committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

# Conflicts:
#	src/main/java/seedu/hustler/game/avatar/Inventory.java
#	src/main/java/seedu/hustler/logic/command/avatar/CheckAvatarCommand.java
#	src/main/java/seedu/hustler/logic/parser/anomaly/AddCommandAnomaly.java

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

nystera committed:
+change findAnomaly

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

jingkang97 committed:
pull

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

ngjiewu committed:
Merge pull request #252 from nystera/master

Added missing documents

ngjiewu committed:
Ignore case sensitive for commands

ngjiewu committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

EnriqueKhai committed:
Update UserGuide.adoc

ngjiewu committed:
Merge pull request #253 from ngjiewu/master

Improved GUI modes and command parsing

nystera committed:
+change Avatar class to make the function adhere to proper OOP principles.

nystera committed:
+changed Avatar class to make the function adhere to proper OOP principles.

nystera committed:
Merge pull request #254 from nystera/master

Made Avatar package more OOP.

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

yzia2000 committed:
fixed RecommededSchedule bugs

yzia2000 committed:
Merge pull request #255 from yzia2000/master

fixed RecommededSchedule bugs

EnriqueKhai committed:
Update UserGuide.adoc

ngjiewu committed:
Updated sort command in UserGuide.adoc

ngjiewu committed:
Updated snooze command summary in UserGuide.adoc

ngjiewu committed:
Fixed addfromlist

ngjiewu committed:
Update addfromlist in UserGuide.adoc

ngjiewu committed:
Fixed case-sensitive commands

ngjiewu committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

ngjiewu committed:
Fixed #267

ngjiewu committed:
Fixed #258, #259, #260

ngjiewu committed:
Fixed #279

ngjiewu committed:
Fixed #257

ngjiewu committed:
Fixed snooze past date

ngjiewu committed:
Fixed SortAnomaly

ngjiewu committed:
Change error msg to DD/MM/YYYY instead of YY

ngjiewu committed:
Merge pull request #285 from ngjiewu/branch-BugFixing

Fixed bugs found in mock PE

jingkang97 committed:
Updated achievements with progress

jingkang97 committed:
Added progress for first login

jingkang97 committed:
Added percentage progress

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

jingkang97 committed:
added log button

jingkang97 committed:
remove history

jingkang97 committed:
Merge pull request #286 from jingkang97/branch-Achievements

[CS2113T-T09-1]-jingkang97 - Added progress for achievement and information to achievements

ngjiewu committed:
Minor UI printing formatting

ngjiewu committed:
Minor formatting to /avatar

EnriqueKhai committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

nystera committed:
Added JUnit testing and removed some old tests that are irrelevant.

nystera committed:
Added JUnit testing for Stats, Level and Avatar.
Modified Stats to be an immutable class.

nystera committed:
Modified ShopList to be immutable.
Purchasable is now an interface that all Items will implement.

nystera committed:
Renamed Purchasable interface to ShopItem.
Removed abstract class ShopItem to make classes more structured and follow OOP principles.

nystera committed:
Made Level and Stats immutable.
Created JUnit testing for Level, Stats, ShopList class.

nystera committed:
fixed some testing bugs. To document.

ngjiewu committed:
Merge pull request #287 from ngjiewu/master

Minor UI printing formatting

ngjiewu committed:
Merge pull request #288 from nystera/master

Created several JUnit tests and changed Command to an interface.

yzia2000 committed:
added setters to Task class

yzia2000 committed:
created Command Design pattern for /edit command

jingkang97 committed:
achievements

yzia2000 committed:
added the edit command

yzia2000 committed:
Merge pull request #290 from yzia2000/master

[CS2113T-T09-1]-yzia2000 - Edit command

yzia2000 committed:
Fixed #270

yzia2000 committed:
fixed #269

jingkang97 committed:
commit

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

jingkang97 committed:
added achievementlist testing

EnriqueKhai committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

EnriqueKhai committed:
fixed undo bug with achievements

EnriqueKhai committed:
fixed redo never off printing

jingkang97 committed:
Completionist testing

EnriqueKhai committed:
fixed timer accepting negative ints

EnriqueKhai committed:
Merge pull request #293 from EnriqueKhai/master

[AY1920S1-CS2113T-T09-1]-nigeltan-fixed bug with undo where achievements total points dont update

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

EnriqueKhai committed:
fixed undo still reloading backup when argument is invalid

EnriqueKhai committed:
Merge pull request #294 from EnriqueKhai/master

fixed undo still reloading backup when argument is invalid

jingkang97 committed:
javadocs

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

jingkang97 committed:
Finish up test for all achievements

jingkang97 committed:
change function name

ngjiewu committed:
Merge pull request #292 from yzia2000/master

[AY1920S1-CS2113T-T09-1]-yzia2000-BugFixes

jingkang97 committed:
checkstyle update 1

jingkang97 committed:
checkstyle update 2

jingkang97 committed:
Merge pull request #297 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-fixed checkstyle and do up achievement testing.

nystera committed:
Changed stats and level to implement an interface.
Todo: Update JUnit testing for Avatar.

nystera committed:
Merge branch 'ClassImmutability'

# Conflicts:
#	src/main/java/seedu/hustler/game/avatar/Stats.java
#	src/test/java/seedu/hustler/game/shop/ShopListTest.java

nystera committed:
Fixed merge bugs.

jingkang97 committed:
Merge pull request #299 from nystera/master

[AY1920S1-CS2113T-T09-1]-nystera-made all game classes immutable.

nystera committed:
Fixed merge bugs.

nystera committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

nystera committed:
Merge pull request #301 from nystera/master

[AY1920S1-CS2113T-T09-1]-nystera-reverted the JUnit testing of ShopList.

jingkang97 committed:
Added Achievement interface

yzia2000 committed:
changed RecommendedSchedule design

yzia2000 committed:
association between Recommended and Schedule

yzia2000 committed:
integrate new RecommendedSchedule with base code

yzia2000 committed:
added anomaly detection to /edit command

EnriqueKhai committed:
bug fixes for undo and redo features

ngjiewu committed:
Merge pull request #302 from yzia2000/master

[AY1920S1-CS2113T-T09-1]-yzia2000-BugFixing

ngjiewu committed:
Merge pull request #303 from EnriqueKhai/master

[AY1920S1-CS2113T-T09-1]-nigeltan-bug fixes for undo and redo features

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

jingkang97 committed:
fixed detect anomalies

jingkang97 committed:
fixed indentation

jingkang97 committed:
Merge pull request #304 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-fixed detect anomalies and checkstyle

ngjiewu committed:
Deleted JavaFX tutorial

ngjiewu committed:
Deleted text-ui-test

ngjiewu committed:
Deleted Setting.java under data

jingkang97 committed:
added achievementlist testing

ngjiewu committed:
Updated build.gradle to run JavaFX on all platforms

ngjiewu committed:
Merge pull request #305 from ngjiewu/master

Fixed jar file and deleted unnecessary files

nystera committed:
Touched up on bugs.
Refurbished AvatarStats test.

nystera committed:
Refurbished AvatarLevel test.

yzia2000 committed:
removed static variables

yzia2000 committed:
Merge branch 'master' of github.com:AY1920S1-CS2113T-T09-1/main

jingkang97 committed:
Edited Completionist test

jingkang97 committed:
First Login test

ngjiewu committed:
Added tests for RecurringTasks

jingkang97 committed:
javadoc for dedicated

ngjiewu committed:
Added test for snooze

ngjiewu committed:
Merge pull request #311 from yzia2000/master

[AY1920S1-CS2113T-T09-1]-yzia2000-BugFixing

jingkang97 committed:
javadoc

jingkang97 committed:
Merge pull request #312 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-Achievement testings

ngjiewu committed:
Added test for sortTask

ngjiewu committed:
Fixed #256

ngjiewu committed:
Merge branch 'master' into branch-Testing

ngjiewu committed:
Merge pull request #313 from ngjiewu/branch-Testing

Added JUnit testing for my implementation and fixed minor bugs

ngjiewu committed:
Renamed to images

ngjiewu committed:
Merge pull request #314 from ngjiewu/master

Renamed to images

ngjiewu committed:
Update README.adoc

yzia2000 committed:
Merge branch 'master' of github.com:AY1920S1-CS2113T-T09-1/main

yzia2000 committed:
added test cases

yzia2000 committed:
added tests to Scheduler, Recommender

yzia2000 committed:
Merge pull request #315 from yzia2000/master

[AY1920S1-CS2113T-T09-1]-yzia2000-Unit Testing

yzia2000 committed:
added checkstyle

yzia2000 committed:
Merge pull request #316 from yzia2000/master

[AY1920S1-CS2113T-T09-1]-yzia2000-CheckStyle

nystera committed:
Update AboutUs.adoc

nystera committed:
Update AboutUs.adoc

nystera committed:
Delete Ui.png

nystera committed:
Add files via upload

nystera committed:
Update README.adoc

nystera committed:
Update AboutUs.adoc

nystera committed:
Update AboutUs.adoc

nystera committed:
Updated javadocs from Game packages.

nystera committed:
Merge branch 'UpdateAvatar'

# Conflicts:
#	src/main/java/seedu/hustler/task/variables/Difficulty.java

nystera committed:
Cleaned up difficulty class.

ngjiewu committed:
Update AboutUs.adoc

nystera committed:
Merge pull request #318 from nystera/master

[AY1920S1-CS2113T-T09-1]-nystera-check styling fix with JUnit testing for game package.

ngjiewu committed:
Fixed sorting tests

ngjiewu committed:
Fixed checkstyle

ngjiewu committed:
Merge pull request #320 from ngjiewu/master

Fixed checkstyle and sorting test

nystera committed:
Added test tasks.

nystera committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

nystera committed:
Added test tasks.

nystera committed:
Merge pull request #322 from nystera/master

Made test tasks.

ngjiewu committed:
Fixed pre-loaded tasks

ngjiewu committed:
Merge pull request #323 from ngjiewu/master

Fixed pre-loaded tasks

ngjiewu committed:
Fixed datetime sorting

ngjiewu committed:
Merge pull request #324 from ngjiewu/master

Fixed datetime sorting

yzia2000 committed:
fixed priority bug in difficulty detection

jingkang97 committed:
Testing

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

jingkang97 committed:
fixed clash bug

jingkang97 committed:
Merge pull request #325 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-fixed bug for clash

yzia2000 committed:
Merge branch 'master' of github.com:AY1920S1-CS2113T-T09-1/main

yzia2000 committed:
Merge pull request #326 from yzia2000/master

[AY1920S1-CS2113T-T09-1]-BugFixes

nystera committed:
Fixed addCommandAnomaly to not take in random sub commands.

jingkang97 committed:
Fixed reminders testing

jingkang97 committed:
Merge pull request #328 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-RemindersTestingBug

EnriqueKhai committed:
writing tests

EnriqueKhai committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

nystera committed:
Fixed checkstyles for task storage, buyanomaly, and inventoryanomaly.

nystera committed:
Merge branch 'fixAddCommandAnomaly'

nystera committed:
Merge branch 'checkStyleFix'

nystera committed:
Fixed small checkstyle issues.

jingkang97 committed:
fixed checkstyle

nystera committed:
Merge pull request #329 from nystera/master

Fixed checkstyle issues for some classes. Fixed addCommandAnomaly to throw proper errors.

jingkang97 committed:
Merge pull request #330 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-fixed checkstyle

EnriqueKhai committed:
checkstyle fixes + mute timer exception

EnriqueKhai committed:
added timer anomaly check for 'timer 1 1 g'

EnriqueKhai committed:
fixed TimerAnomaly.java checkstyle issues

EnriqueKhai committed:
timeranomaly checkstyle

EnriqueKhai committed:
Merge pull request #331 from EnriqueKhai/master

[CS2113T-T09-1]-nigeltan-fixedCheckStyle

EnriqueKhai committed:
fix merge compile fail

jingkang97 committed:
Added @@author

EnriqueKhai committed:
Merge pull request #332 from EnriqueKhai/master

fix merge compile fail

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

EnriqueKhai committed:
Rename UIMessages.java to UiMessages.java

EnriqueKhai committed:
Merge pull request #333 from EnriqueKhai/master

Rename UIMessages.java to UiMessages.java

EnriqueKhai committed:
Rename statusTypes.java to StatusTypes.java

EnriqueKhai committed:
Merge pull request #334 from EnriqueKhai/master

Rename statusTypes.java to StatusTypes.java

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

ngjiewu committed:
Added missing javadoc

ngjiewu committed:
Fixed checkstyle

ngjiewu committed:
Merge pull request #335 from ngjiewu/master

Fixed small checkstyle error

yzia2000 committed:
edited storage test case

yzia2000 committed:
Merge branch 'master' of github.com:AY1920S1-CS2113T-T09-1/main

yzia2000 committed:
Merge pull request #336 from yzia2000/master

TaskStorageTest fix

EnriqueKhai committed:
undo fixes

EnriqueKhai committed:
sync

ngjiewu committed:
Update AboutUs.adoc

EnriqueKhai committed:
checkstyle

jingkang97 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main into branch-Achievements

nystera committed:
Fixed test checkstyle issues.

nystera committed:
Merge pull request #337 from EnriqueKhai/master

undo fixes

nystera committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-T09-1/main

nystera committed:
Merge pull request #338 from nystera/master

Fixed checkstyle errors for JUNIT testing

jingkang97 committed:
Fixed checkstyle for achievement

jingkang97 committed:
Merge pull request #339 from jingkang97/branch-Achievements

[AY1920S1-CS2113T-T09-1]-jingkang97-Fixed checkstyle for reminders

jingkang97 committed:
Fixed reminders checkstyle

jingkang97 committed:
Merge pull request #340 from jingkang97/branch-Achievements

Fixed reminders checkstyle

jingkang97 committed:
fixed comments

jingkang97 committed:
Merge pull request #341 from jingkang97/branch-Achievements

fixed comments

EnriqueKhai committed:
final checkstyle fix

EnriqueKhai committed:
Merge pull request #342 from EnriqueKhai/master

final checkstyle fix

yzia2000 committed:
fixed checkstyle

yzia2000 committed:
Merge pull request #343 from yzia2000/master

fixed checkstyle

yzia2000 committed:
fixed bug for 20 cases recommended schedule

yzia2000 committed:
Merge pull request #345 from yzia2000/master

Fixed Recommended Schedule bug for test cases
